src/app/new-character/new-character.component.ts

```diff
@@ -25,6 +25,17 @@ export class NewCharacterComponent extends CharacterAbstractComponent {
 
   statPoints: string = 'Custom'
   pointOptions: string[] = ['Custom', '10', '15', '20', '25']
+  feet = 0
+  inches = 0
+  totalInches = signal(0)
+
+  updateTotalInches () {
+    const total = (this.feet * 12) + this.inches
+    this.totalInches.set(total)
+    const character = this.character()
+    character.height = total
+    this.updateCharacter()
+  }
```

Comment:
Good attempt to use signals. I've been wondering if we could leverage that in more places in the app.

However if we're going to use signals, feet and inches should be signals and totalInches should be a computed signal.
https://angular.dev/guide/signals